scripts/environment.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftCrypto open source project
+##
+## Copyright (c) 2020 Apple Inc. and the SwiftCrypto project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.md for the list of SwiftCrypto project authors
```

Comment:
This needs to be `CONTRIBUTORS.txt` temporarily to get through CI. If we land this first, we can update #24 to change this.

Reply:
haha right, got it, I was unsure about that at first, thanks for pointing it out, updating now.